internal/routes/all_routes.go

```diff
@@ -71,11 +71,11 @@ func MountAPI(r *chi.Mux) {
 
 				// TODO DEPRECATED: move this to outside of /sources (see below)
 				r.Get("/instance_types", s.ListInstanceTypes)
+				r.With(middleware.Pagination).Get("/launch_templates", s.ListLaunchTemplates)
 
 				// TODO DEPRECATED: replaced with upload_info
 				r.Get("/account_identity", s.GetAWSAccountIdentity)
 
-				r.Get("/launch_templates", s.ListLaunchTemplates)
 				r.Get("/upload_info", s.GetSourceUploadInfo)
 				r.Route("/validate_permissions", func(r chi.Router) {
 					r.Get("/", s.ValidatePermissions)
```

Comment:
If you get to rebasing this once again, can you move the line back to line 78? Because there is another PR which is removing the deprecated instance_types endpoint and this will end up in conflict. The line move done here does not seem to be useful.